<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrControlVoltage_Unity</class>
 <widget class="QWidget" name="CtrControlVoltage_Unity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
border:1px solid transparent;
}</string>
  </property>
  <widget class="QLabel" name="LBLredline2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>70</y>
     <width>50</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px solid red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLredline1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>50</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px solid red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>本车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLValueRect2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>40</y>
     <width>49</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>21</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:10px;</string>
   </property>
   <property name="text">
    <string>[V]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBackground">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>39</y>
     <width>51</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLValue">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font:14px;</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLBackground2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>39</y>
     <width>51</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>控制电压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_36">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLValueRect">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>40</y>
     <width>49</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLyellowline2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>100</y>
     <width>50</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px solid yellow;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLyellowline1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>50</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px solid yellow;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>它车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>40</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLValue2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>20</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font:14px;</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L1_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>175</y>
     <width>10</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>20</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;color:white</string>
   </property>
   <property name="text">
    <string>120</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>L1</zorder>
  <zorder>L1_9</zorder>
  <zorder>label</zorder>
  <zorder>label_9</zorder>
  <zorder>label_14</zorder>
  <zorder>LBLBackground</zorder>
  <zorder>label_2</zorder>
  <zorder>LBLValue</zorder>
  <zorder>L1_17</zorder>
  <zorder>label_17</zorder>
  <zorder>L1_36</zorder>
  <zorder>LBLValueRect</zorder>
  <zorder>label_15</zorder>
  <zorder>LBLyellowline1</zorder>
  <zorder>label_13</zorder>
  <zorder>L1_12</zorder>
  <zorder>L1_13</zorder>
  <zorder>label_16</zorder>
  <zorder>label_4</zorder>
  <zorder>LBLValue2</zorder>
  <zorder>L1_15</zorder>
  <zorder>L1_14</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>LBLredline1</zorder>
  <zorder>LBLBackground2</zorder>
  <zorder>LBLValueRect2</zorder>
  <zorder>LBLredline2</zorder>
  <zorder>LBLyellowline2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
